ui-grid/ui-grid.d.ts

```diff
@@ -1512,6 +1512,10 @@ declare namespace uiGrid {
              */
             expandableRowHeight?: number;
             /**
+             * reference to the parent grid scope (the parent scope of the sub-grid element)
+             */
+            expandableRowScope?: Object;
```

Comment:
Type of `Object` lgtm since that's what is being used in app scope provider property.  The only suggestion would be to perhaps consider matching the full type annotation used there: `appScopeProvider?: ng.IScope | Object;`